### PR TITLE
 DeFi Score Chinese whitepaper

### PR DESCRIPTION
**Pull request by anthonyhuanggr**
Hi All,

DeFi Score theory is great and I translated the whitepaper into Chinese and share with Chinese audience.
Please review and any issue needs to assist, please let me know I will follow up.
Many thanks.

Anthony

### CODE REVIEW

**Comment by jclancy93** [comment]
谢谢 @anthonyhuanggr!! 我会说一点中文但是我的中文水平不太高。

I've asked @imjerryyang to review the PR. We will get his merged ASAP. Thanks again 🙂

File: whitepaper-zh.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+# DeFi评估：评估无许可贷款协议的风险 <br/>  
+本文介绍一个无许可贷款协议（Permissionless Lending Protocols）中风险水平的评估模型。为了展现这些协议平台中存在的各种风险，我们使用了一个多因素模型来研究智能合约、抵押品和流动性风险。该模型混合了公链中链上和链外的数据来更好地评估不同的、基于以太坊的无许可贷款协议产品的相对风险水平。<br><br/>
```
Comment (imjerryyang) [inline whitepaper-zh.md]: off-chain 通常被翻译为链下（Very Minor）

File: whitepaper-zh.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+为了更好地理解和建模DeFi领域中的风险将是促使其走向成熟最重要的一步。有许多工作要做，但现在是开始的时候了。希望这个模型和其他类似的模型能够为DeFi进行稳健的风险评估奠定基础。<br><br>
+在这里，我们引入了一个定量模型来评估这种无许可贷款协议（Permissionless lending protocols）中的风险水平。<br><br>
+https://www.blockchain.com/btc/tx/4a5e1e4baab89f3a32518a88c31bc87f618f76673e2cc77ab2127b7afdeda33b<br>
+根据DeFi Pulse贷款类目中总锁定价值（TVL）数据 <br><br>
+## 无许可协议概况<br><br>
+无许可贷款协议是允许用户通过以太坊的“智能合约”借贷各种不同数字资产的系统。这些“智能合约”并不是法律意义上的合约；而是有效运行在以太坊链上的计算机代码。借出资产的用户将从其贷款中获得的利息。借款人提供抵押品（通常高于借款价值），并支付可变利息。无许可贷款协议的一些项目包括compound、dydx和nuo。<br><br>
+### 智能合约风险<br><br>
+在DeFi中，智能合约的风险是主要风险。尽管DeFi通常被称为去信任的，DeFi平台的用户必须信任他们正在交互的智能合约，这意味着用户信任合约代码就像信任Web2.0的应用那样。但智能合约对用户来说可能是不透明/磨糊的。还有一种风险是，智能合约由于不安全而遭到黑客攻击，这对合约中的用户产生严重的财务影响，包括，失去锁定在合约中的抵押品。我们提出的模型着眼于评估智能合约风险的这两个要素。<br><br>
+### 代码安全<br><br>
+当用户在与DeFi平台交互时存在丢失锁定在智能合约中的资金风险时，智能合约的安全性就显得非常重要。正如生态系统所得到的教训那样，智能合约中的错误可能导致重大的财务损失。例如，“DAO攻击”是针对“DAO”的项目发起的攻击，“DAO”是去中心自治组织的一种形式，即以太坊链上智能合约中体现的“虚拟”组织。2016年7月17日，一个未知的个人或团体从DAO快速转移大量ETH，导致大约360万ETH- DAO的1/3 ETH从DAO的地址转移到攻击者控制的地址。<br><br>
```
Comment (imjerryyang) [inline whitepaper-zh.md]: (Missing a '-' )导致大约360万ETH - DAO的1/3 ETH -

File: whitepaper-zh.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+中本聪创立比特币的主要动力是2008年的金融危机。他在比特币的创世区块（Genesis Block）提到了当时的银行救助计划。一些人认为，金融危机部分原因是由于金融界对整个金融体系信贷风险的误解和错误定价造成的。风险管理是现代金融中最关键的部分之一，它为我们建立了一个更加稳健和安全的金融体系。<br><br>
+在过去的一年里，我们看到了以太坊上无许可的金融协议爆炸式增长，其中很大一部分集中在借贷市场。这些市场已发展成为“去中心化金融”或“DeFi”的最大子类别，年增长率达355%。然而，并非所有的借贷平台都是一样的。不同的无许可贷款产品具有非常不同的风险/回报，这使得仅比较它们的利率就好像比较苹果和桔子一样毫无意义。<br><br>
+为了更好地理解和建模DeFi领域中的风险将是促使其走向成熟最重要的一步。有许多工作要做，但现在是开始的时候了。希望这个模型和其他类似的模型能够为DeFi进行稳健的风险评估奠定基础。<br><br>
+在这里，我们引入了一个定量模型来评估这种无许可贷款协议（Permissionless lending protocols）中的风险水平。<br><br>
+https://www.blockchain.com/btc/tx/4a5e1e4baab89f3a32518a88c31bc87f618f76673e2cc77ab2127b7afdeda33b<br>
+根据DeFi Pulse贷款类目中总锁定价值（TVL）数据 <br><br>
+## 无许可协议概况<br><br>
+无许可贷款协议是允许用户通过以太坊的“智能合约”借贷各种不同数字资产的系统。这些“智能合约”并不是法律意义上的合约；而是有效运行在以太坊链上的计算机代码。借出资产的用户将从其贷款中获得的利息。借款人提供抵押品（通常高于借款价值），并支付可变利息。无许可贷款协议的一些项目包括compound、dydx和nuo。<br><br>
+### 智能合约风险<br><br>
+在DeFi中，智能合约的风险是主要风险。尽管DeFi通常被称为去信任的，DeFi平台的用户必须信任他们正在交互的智能合约，这意味着用户信任合约代码就像信任Web2.0的应用那样。但智能合约对用户来说可能是不透明/磨糊的。还有一种风险是，智能合约由于不安全而遭到黑客攻击，这对合约中的用户产生严重的财务影响，包括，失去锁定在合约中的抵押品。我们提出的模型着眼于评估智能合约风险的这两个要素。<br><br>
```
Comment (imjerryyang) [inline whitepaper-zh.md]: 感觉后面加上这个单词原词稍微好一点：尽管DeFi通常被称为去信任的(Trustless)

File: whitepaper-zh.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
+# DeFi评估：评估无许可贷款协议的风险 <br/>  
+本文介绍一个无许可贷款协议（Permissionless Lending Protocols）中风险水平的评估模型。为了展现这些协议平台中存在的各种风险，我们使用了一个多因素模型来研究智能合约、抵押品和流动性风险。该模型混合了公链中链上和链外的数据来更好地评估不同的、基于以太坊的无许可贷款协议产品的相对风险水平。<br><br/>
+## 简介 <br/>
+中本聪创立比特币的主要动力是2008年的金融危机。他在比特币的创世区块（Genesis Block）提到了当时的银行救助计划。一些人认为，金融危机部分原因是由于金融界对整个金融体系信贷风险的误解和错误定价造成的。风险管理是现代金融中最关键的部分之一，它为我们建立了一个更加稳健和安全的金融体系。<br><br>
+在过去的一年里，我们看到了以太坊上无许可的金融协议爆炸式增长，其中很大一部分集中在借贷市场。这些市场已发展成为“去中心化金融”或“DeFi”的最大子类别，年增长率达355%。然而，并非所有的借贷平台都是一样的。不同的无许可贷款产品具有非常不同的风险/回报，这使得仅比较它们的利率就好像比较苹果和桔子一样毫无意义。<br><br>
+为了更好地理解和建模DeFi领域中的风险将是促使其走向成熟最重要的一步。有许多工作要做，但现在是开始的时候了。希望这个模型和其他类似的模型能够为DeFi进行稳健的风险评估奠定基础。<br><br>
+在这里，我们引入了一个定量模型来评估这种无许可贷款协议（Permissionless lending protocols）中的风险水平。<br><br>
+https://www.blockchain.com/btc/tx/4a5e1e4baab89f3a32518a88c31bc87f618f76673e2cc77ab2127b7afdeda33b<br>
```
Comment (imjerryyang) [inline whitepaper-zh.md]: 这是注释可能需要表明, Annotations here

File: whitepaper-zh.md [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+根据DeFi Pulse贷款类目中总锁定价值（TVL）数据 <br><br>
+## 无许可协议概况<br><br>
+无许可贷款协议是允许用户通过以太坊的“智能合约”借贷各种不同数字资产的系统。这些“智能合约”并不是法律意义上的合约；而是有效运行在以太坊链上的计算机代码。借出资产的用户将从其贷款中获得的利息。借款人提供抵押品（通常高于借款价值），并支付可变利息。无许可贷款协议的一些项目包括compound、dydx和nuo。<br><br>
+### 智能合约风险<br><br>
+在DeFi中，智能合约的风险是主要风险。尽管DeFi通常被称为去信任的，DeFi平台的用户必须信任他们正在交互的智能合约，这意味着用户信任合约代码就像信任Web2.0的应用那样。但智能合约对用户来说可能是不透明/磨糊的。还有一种风险是，智能合约由于不安全而遭到黑客攻击，这对合约中的用户产生严重的财务影响，包括，失去锁定在合约中的抵押品。我们提出的模型着眼于评估智能合约风险的这两个要素。<br><br>
+### 代码安全<br><br>
+当用户在与DeFi平台交互时存在丢失锁定在智能合约中的资金风险时，智能合约的安全性就显得非常重要。正如生态系统所得到的教训那样，智能合约中的错误可能导致重大的财务损失。例如，“DAO攻击”是针对“DAO”的项目发起的攻击，“DAO”是去中心自治组织的一种形式，即以太坊链上智能合约中体现的“虚拟”组织。2016年7月17日，一个未知的个人或团体从DAO快速转移大量ETH，导致大约360万ETH- DAO的1/3 ETH从DAO的地址转移到攻击者控制的地址。<br><br>
+虽然不能保证任何智能合约都是安全的、没有错误的，但一个信誉良好的安全公司提供的彻底的代码审计和形式化的验证过程将有助于发现严重的错误。缺陷赏金计划（Bug bounty programs）是另一个积极的体现，表明开发团队对安全的重视，鼓励独立的安全研究人员发现协议代码缺陷，最终允许更广泛的安全审查监督。<br><br>
```
Comment (imjerryyang) [inline whitepaper-zh.md]: Formal Verification 感觉应该翻译成 “正规化的验证 ” or “正式验证” 比较恰当。

File: whitepaper-zh.md [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+我们的模型通过审查链外但公开的数据来评估代码的安全性：<br>
+1.	代码审计：代码是否经过信誉良好的安全团队（consensys diligence，trail of bits，others）的审计？<br>
+2.	形式验证：代码是否已经由信誉良好的安全团队形式化的验证。<br>
+3.	赏金计划：开发团队是否提供公开详细的代码缺陷赏金计划。<br><br>
+Securities and Exchange Commission, Release No. 81207, https://www.sec.gov/litigation/investreport/34‐81207.pdf (“SEC DAO Report”) at 1.<br><br>
```
Comment (imjerryyang) [inline whitepaper-zh.md]: It is an annotation.

File: whitepaper-zh.md [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+DAO攻击中资金是通过以太坊区块链（ID）的一个特别的“分叉”收回的，但这种分叉不能指望能从未来的黑客行为中收回。<br><br>
```
Comment (imjerryyang) [inline whitepaper-zh.md]: ditto

File: whitepaper-zh.md [hunk 8]
```diff
@@ -0,0 +1,80 @@
...
+### 代码开放<br><br>
+DeFi承诺的一部分是，智能合约是完全在链上的，这意味着代码是可验证和透明的。虽然DeFi平台的开发人员仍然能够以各种方式隐藏他们的代码，例如不验证字节码和使用链外预言机（oracles）。这种磨糊的安全性只能提供薄弱的安全保证，最坏的情况是会导致关键错误的被延迟找到。虽然字节码反编译是可能的，但这是一个困难和耗时的过程，并且很难遵循“不相信，只验证”的口号。<br><br>
+代码开放性是通过审查链外的公共数据，来验证字节码是否被验证来评估。<br><br>
+## 财务风险<br><br>
+DeFi包含许多与传统金融相同的风险。虽然大多数贷款平台使用了超额抵押来降低信贷风险，但超额抵押并不能完全消除这样的无许可信贷的风险。因为我们都知道，现在的密码/数字/加密资产是极不稳定的，这些平台没能够拿这些不稳定的抵押资产来全部对冲借款。<br><br>
+目前我们的模型着眼于金融风险的两个要素：<br><br>
+### 抵押品<br><br>
+如果我们没能识别和接受的链上相对应借款人的声誉或身份，那么在DeFi借贷市场上避免信贷风险的唯一方法是使用超额抵押。虽然目前所有平台都使用了这种非常保守的担保，但密码/数字/加密资产的高波动性意味着，这些超额担保仍然不足。<br><br>
+这些DeFi平台支持的抵押资产也有很大的差异，其中一些由流动性更高、更稳定的资产组成。例如，一个平台接受ETH抵押。虽然ETH仍然不是一种非常稳定的资产，但与link6这样的资产相比，它相对稳定且流动性高。在考虑平台风险时，这些抵押品的差异是一个重要因素。<br><br>
```
Comment (imjerryyang) [inline whitepaper-zh.md]: 6 is an annotation, 6是注释角标

**Comment by imjerryyang** [comment]
> 谢谢 @anthonyhuanggr!! 我会说一点中文但是我的中文水平不太高。
> 
> I've asked @imjerryyang to review the PR. We will get his merged ASAP. Thanks again 🙂

I did a quick review. Thanks @anthonyhuanggr, it is a very good translation in general, just little formate issues need to be fixed.

**Comment by anthonyhuanggr** [comment]
Thanks @jclancy93 

Hi @imjerryyang 
Thank you for your time to review, I fixed the issue accordingly, please have a review again if your time available, Many thanks.

**Comment by imjerryyang** [comment]
Looks good to me! Thanks a lot @anthonyhuanggr

**Comment by jclancy93** [comment]
@all-contributors please add @ anthonyhuanggr for translation

**Comment by jclancy93** [comment]
@all-contributors please add @anthonyhuanggr for translation

**Pull request merged**